### PR TITLE
Add military-related brands to forbidden categories

### PR DESCRIPTION
**Pull request by uncenter**
Follow up to #11097.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ We welcome icon requests. Before you submit a new issue please make sure the ico
     - Governmental agencies, programs, departments
        - Allowed: International organizations and NGOs with supranational interests
        - Allowed: Space agencies
```
Comment (uncenter) [inline CONTRIBUTING.md:60]: Trying to figure out how we can explicitly mention that "no military brands/orgs" overrules the allowed "international organizations and NGOs with supranational interests"...

Comment (mondeja) [reply CONTRIBUTING.md:60]: Note that "International organizations and NGOs with supranational interests" is mentioned in the context of governmental agencies. Organizations like the NATO seems to me that doesn't resides in that category.

Comment (adamrusted) [reply CONTRIBUTING.md:60]: Totally agree, @mondeja though that does leave stuff like NATO in a grey area - and I think we're best to explicitly state we will not accept military-affiliated organisations.

Comment (mondeja) [reply CONTRIBUTING.md:60]: Yes, I agree. Just trying to state that doesn't overrules the previous "governmental" category, that could be a different category. I'm fine with the current wording but not being a native English speaker, let to you the approval.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -58,6 +58,7 @@ We welcome icon requests. Before you submit a new issue please make sure the ico
...
+    - Military-related brands and organizations
```
Comment (adamrusted) [inline CONTRIBUTING.md:61]:
```suggestion
    - Military-related brands and organizations
    - Governmental agencies, programs, departments
       - Allowed: International organizations and NGOs with supranational interests
       - Allowed: Space agencies
```

I'd move military above the government bit - just to be explicit about it earlier.

**Review by adamrusted — APPROVED** [review]

**Pull request merged**